### DISCUSSION TITLE
I cant understand how search by nested [Jun 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2502] [Discord Threads]
> 🙋‍♂️ **unkn**

**Comment by o1lab** [comment]
[Autogenerated from discord](https://discord.com/channels/661905455894888490/988480017874952273)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
